### ISSUE TITLE
Italic bits not set for Noto Sans Italic and BoldItalic

### ISSUE THREAD

**Issue by moontypespace** [googlefonts/noto-fonts#2269]
### Title 
Italic bits not set for Noto Sans Italic and Bold Italic

### Font 
NotoSans-BoldItalic.ttf
NotoSans-Italic.ttf

### Where the font came from, and when
https://github.com/googlefonts/noto-fonts/tree/main/hinted/ttf/NotoSans
Date: 2022-03-03 (preferred format)

### Font Version
2.007

### Issue
The Italic bits are not set in:
- 'OS/2' table fsSelection
- 'head' table macStyle

**Comment by simoncozens** [comment]
Traced back to glyphsLib doing this:

```xml
<instance name="Noto Sans Italic"
  familyname="Noto Sans"
  stylename="Italic"
  filename="../instance_ufo/NotoSans-Italic.ufo"
  stylemapfamilyname="Noto Sans Italic"
  stylemapstylename="regular">
```

(Bogus stylemap family name / style name.) Now to work out why glyphsLib is doing that for this font in particular...

**Comment by simoncozens** [comment]
Found it, it's in the style linking flags. PR soon.

**Comment by moontypespace** [comment]
@simoncozens Thank you very much for having a look into it. I much appreciate you fast contribution. May I ask you, when there will be new fixed fonts available?

**Comment by marekjez86** [comment]
soon on GitHub noto-fonts; within the next few days

Other distribution points will take longer

**Comment by moontypespace** [comment]
Awesome. Thank you very much.

**Comment by xctvcd** [comment]
Hi @marekjez86, thanks for the quick fix, there’s an issue left, see #2279. Cheers!

**Comment by simoncozens** [comment]
I will take a look at that next week.
